net-dns/getdns/getdns-1.5.0.ebuild

```diff
@@ -29,8 +29,12 @@ DEPEND="
 	libev? ( dev-libs/libev:= )
 	libuv? ( dev-libs/libuv:= )
 "
+DEPEND="
+	${COMMON_DEPEND}
+	doc? ( app-doc/doxygen )
```

Comment:
this (`app-doc/doxygen`) should more appropriately be in BDEPEND as it is executed
on the build host.

Reply:
I fixed that.